web/sites/default/config/views.view.council_documents.yml

```diff
@@ -919,6 +906,299 @@ display:
         - 'config:field.storage.node.field_full_document_title'
         - 'config:field.storage.node.field_introduced_by'
         - 'config:field.storage.node.field_is_emergency_ordinance'
+  block_council_documents_by_committee:
```

Comment:
Could we hide this block on the main City Council page? (essentially, a filter of Committee is not City Council).

I think we will only want this block to show on the committee pages since city council will always have a bunch of documents assigned to it and we intend on creating a separate public facing page for that.

Reply:
Added a taxonomy relationship and a filter for all committees except "City Council".